src/components/learner-credit-management/data/constants.js

```diff
@@ -85,3 +121,6 @@ export const learnerCreditManagementQueryKeys = {
   budgetGroupLearners: (budgetId) => [...learnerCreditManagementQueryKeys.budget(budgetId), 'group learners'],
   enterpriseCustomer: (enterpriseId) => [...learnerCreditManagementQueryKeys.all, 'enterpriseCustomer', enterpriseId],
 };
+
+// Route to learner credit
+export const LEARNER_CREDIT_ROUTE = '/:enterpriseSlug/admin/:enterpriseAppPage/:budgetId/:activeTabKey';
```

Comment:
[nit/sanity check] Is the `:activeTabKey` always present for this route?

```jsx
<Route
  path="/:budgetId/:activeTabKey?"
  element={<BudgetDetailPage />}
/>
```

Based on this, we might to define the route as having parity with the upstream route definition? That said, when we actually _use_ this constant, we are always passing the `activeTabKey` route param, so this is largely a moot point anyways.

Still might be good to have parity with the route definition, though?

Reply:
Updated to keep in parity. Great callout 👍🏽